deploy-on-k8s.yml

```diff
@@ -13,14 +13,18 @@ spec:
       labels:
         app: ci
     spec:
+      securityContext:
+        runAsUser: 65534
+        runAsGroup: 65534
+      serviceAccountName: builder
```

Comment:
This serviceAccount is automatically created by openshift in each project namespace.